notebooks/05_Kubeflow_Pipeline/src/Pipeline_mnist/launcher/src/launch_tfjob.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2019 kubeflow.org.
```

Comment:
Do you write this file from scratch or it comes from some community source?  
My point is if that's generic enough, consider to file PR back to pipeline project. 
If some codes are from community, any reason to have a copy here? Does tfjob launch providers component?

Reply:
You're right, I can simply download this python file in `Dockerfile` rather than having a copy.